paseo-system-parachain/asset-hub/active/example.yml

```diff
@@ -0,0 +1,23 @@
+# Identity to be set up for all system chain collators running by Example
+identity:
+  display: Example
+  email: hi@example.io
+  website: https://example.io
+  twitter: @example
+  discord: example
+  riot: @example:matrix.org
+
+system_chain: AssetHub
+
+accounts:
+  # Here example is running two collators, 0 and 1
+  0:
+    identity:
+      display: Example-01
+    account_id: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+    aura_id: 0x78a70e070479190982e8d28ff0c038c1f20690619576525d48312c2cbdb1f3ed
```

Comment:
If we want to launch a mirror of what is running in Polkadot, we might want to ask for `AssetHubPolkadotAuraId` keys for AssetHub :thinking:

But..maybe we don't want that.